src/zsh_completion/_firejail.in

```diff
@@ -127,6 +127,7 @@ _firejail_args=(
     '--nosound[disable sound system]'
     '--nou2f[disable U2F devices]'
     '--novideo[disable video devices]'
+    '--keep-shell-rc[disable skeleton files]'
```

Comment:
The options are sorted alphabetically in this file; move it below `--keep-dev-shm`.